### PR TITLE
EZQMS-394: Update diff viewer lint button colors

### PR DESCRIPTION
**Pull request by annano**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svele-check is applied (rush svelte check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

File: packages/theme/styles/_text-editor.scss [hunk 1]
```diff
@@ -283,10 +283,10 @@
   line-height: 1.1em;
 
   &.add {
-    background: lightblue;
+    background: var(--text-editor-highlighted-node-add-background-color);
```
Comment (aonnikov) [inline packages/theme/styles/_text-editor.scss:286]: can we change these variables to be more readable?

Comment (annano) [reply packages/theme/styles/_text-editor.scss:286]: How is it not readable? What exactly do you want to rename?

Comment (aonnikov) [reply packages/theme/styles/_text-editor.scss:286]: `highlighted-node` is specific for node highlight plugin, and it is reflected in the variable name. I don't like that other parts of editor know about this plugin at all.

Comment (annano) [reply packages/theme/styles/_text-editor.scss:286]: `highlighted node` here does not mean it is part of highlighted node plugin, it literally means that the node is highlighted.

**Review by aonnikov — APPROVED** [review]

**Pull request merged**